src/App.js

```diff
@@ -1,24 +1,129 @@
-import logo from './logo.svg';
+import { useState, useEffect } from 'react';
 import './App.css';
+import './styles.css';
+
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <>
+    <NavBar></NavBar>
+
+    <div className="friendstop">
+      <ul>
+        <h1>FRIENDS</h1>
+        <input type="text" placeholder="Search by username..."></input>
+      </ul>
+    </div>
+      <div className="friendrequestoutercontainer">
+        <div className="friendrequestinnercontainer">
+        <h1>Friend Requests</h1>
+          <ul>
+            <FriendRequest name="Konrad Radecki" username="the1krad_" />
```

Comment:
instead of hardcoding the `FriendRequest`s here maybe move these into a state variable within the `App` Component like:
`const [friendRequests, setFriendRequests] = useState([
    { name: 'Konrad Radecki', username: 'the1krad_' },
    { name: 'Finn McMillan', username: 'finnmcm_' }
  ]);`

Reply:
This would allow you add and update this list easily when required..Question: How would you iterate over this state variable? hint: .map